tests/test_acasclient.py

```diff
@@ -2719,56 +2721,115 @@ def can_delete_lot(self, user_client, lot_corp_name, set_owner_first=True):
             self.assertTrue(response['success'])
             return True
 
-        # Global compound, with experiment in Global project
-        ## Deny Rule: Not an acas user so can't delete because dependent experiment exists
+        def check_lot_exists_in_experiment(self, lot_corp_name, experiment_code_name):
+            experiment = self.client.get_experiment_by_code(experiment_code_name, full = True)
+            # when experiment is deleted, the analysis groups for the lot are deleted
+            for analysis_group in experiment["analysisGroups"]:
+                if analysis_group["deleted"] == True:
```

Comment:
Should we be checking `ignored` here too?

Reply:
Fixed thanks.